meilisearch/tests/index/create_index.rs

```diff
@@ -17,7 +17,7 @@ async fn create_index_no_primary_key() {
 
     assert_eq!(response["status"], "enqueued");
 
-    let response = index.wait_task(0).await;
+    let response = index.wait_task(response.uid()).await;
```

Comment:
Not related to this line, but above there is a:
```rust
let server = Server::new().await;
let index = server.index("test");
```

That can be replaced with a:
```rust
    let server = Server::new_shared().await;
    let index = server.unique_index();
```